hack/jenkins/upload-to-gcs.sh

```diff
@@ -116,20 +116,22 @@ function find_version() {
     cat version
   elif [[ -e "hack/lib/version.sh" ]]; then
     export KUBE_ROOT="."
+    # shellcheck disable=SC1091
     source "hack/lib/version.sh"
     kube::version::get_version_vars
     echo "${KUBE_GIT_VERSION-}"
   else
     # Last resort from the started.json
-    gsutil cat ${gcs_build_path}/started.json 2>/dev/null |\
+    gsutil cat "${gcs_build_path}/started.json" 2>/dev/null |\
      sed -n 's/ *"version": *"\([^"]*\)",*/\1/p'
   fi
   )
 }
 
 # Output started.json. Use test function below!
 function print_started() {
-  local metadata_keys=$(compgen -e | grep ^BUILD_METADATA_)
+  metadata_keys=$(compgen -e | grep ^BUILD_METADATA_)
+  local metadata_keys
```

Comment:
Why did it change and what did it fix:
```
In ./hack/jenkins/upload-to-gcs.sh line 132:
  local metadata_keys=$(compgen -e | grep ^BUILD_METADATA_)
        ^-----------^ SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
local has to come before assignment

Reply:
I have some questions about it. According to SC2155, I should have no problem to modify it like this.

Reply:
no, it needs to be first:
https://github.com/koalaman/shellcheck/wiki/SC2155#correct-code-1

the reason is `local` creates a variable with a reduced scope, the scope cannot be reduced retroactively by merely re-declaring it.

Reply:
this was fixed, thanks! resolving